### DISCUSSION TITLE
Does this extension work with firefox nightly for android?

### DISCUSSION THREAD

**Discussion by Dark-Dragon** [younesaassila/ttv-lol-pro#139] [Q&A]
I'm looking to effectively block twitch ads on mobile and from what I can tell there's a fair chance this extension would in theory work with firefox nightly for android. Before I look into how to install unlisted/unsigned extensions though, I wouldn't mind hearing if others have tried this.

Alternative ways to block twitch ads effectively on an android device are also appreciated.

**Comment by zGato** [comment]
Hello @Dark-Dragon 

You either have the option to install a modded application for Twitch or use PAC files (only possible via WiFi)

**Comment by younesaassila** [comment]
The v1 extension is signed and should work on Android. It's effectiveness varies by user though.
v2 might be compatible but you'd have to do some Googling to figure out how to install an unsigned extension (if it is even possible?)

**Comment by MisterSheeple** [comment]
Yes, v1 works, but in my opinion you'll get a better experience by using a modded Twitch app (as zGato suggested) or a third party client such as Xtra, which has ad blocking built in via TTV LOL's proxy.

**Comment by kojitsari** [comment]
Proxy Droid works to utilize the v2 proxies but you need to remember to disable all other adblock features. Or you can use Xtra's "Custom Proxy URL" feature with the alternative ttv-lol-pro v1 proxies and it covers most ads (some slip through). I haven't been able to get the v2 proxies to work with Xtra yet.
